modules/Bio/EnsEMBL/Compara/RunnableDB/EpoLowCoverage/FindPairwiseMlssLocation.pm

```diff
@@ -212,4 +216,76 @@ sub _optimal_aln_for_genome_db {
     return [$all_alns_for_gdb->{$best_ref_gdb_id}->{compara_db}, $best_aln_mlss_id, $best_ref_gdb_id];
 }
 
+sub _get_msa_coverage_stats {
+    my ( $self, $ref_gdbs ) = @_;
+    
+    # if all references have genome_coverage and genome_length tags in the current epo_db, then we will use that
```

Comment:
It can't in the new merged mode. But it can if you're running EPO2x separately. It then checks the previous EPO db for the tags if they don't exist in `epo_db` (as I've explained in point 4 of my initial PR comment).

Reply:
I'm expecting more than that: the whole for loop

Reply:
I know the principle, but that's not the explanation I was looking for. The answer is in fact that this doesn't come from `$self->compara_dba` (which doesn't have any species-tree node tags) but `$self->param('base_location')` which is the current EPO database